### PR TITLE
(maint) merge up 5.3.x to master

### PR DESCRIPTION
**Pull request by Iristyle**

### CODE REVIEW

File: configs/components/facter.rb [hunk 1]
```diff
@@ -19,7 +19,11 @@
   pkg.build_requires 'leatherman'
   pkg.build_requires 'runtime'
   pkg.build_requires 'cpp-hocon'
-  pkg.build_requires 'libwhereami'
+
+  unless platform.is_aix? || platform.is_solaris?
+    # libwhereami doesn't support AIX or Solaris yet
+    pkg.build_requires "libwhereami"
+  end
```
Comment (caseywilliams) [inline configs/components/facter.rb]: Master should actually have libwhereami - its master includes support for solaris/aix

**Comment by McdonaldSeanp** [comment]
👍

File: configs/components/facter.rb (hunk 1 above)
Comment (caseywilliams) [reply configs/components/facter.rb]: (Other than that these changes look fine to me)

Comment (Iristyle) [reply configs/components/facter.rb]: So master should be completely unguarded then, like?

```ruby
pkg.build_requires 'libwhereami'
```

Comment (caseywilliams) [reply configs/components/facter.rb]: Right, just include for all platforms

**Comment by ScottGarman** [comment]
:+1:

**Pull request merged**